codex/blockexchange/protobuf/message.nim

```diff
@@ -29,6 +29,7 @@ type
     cancel*: bool           # Whether this revokes an entry
     wantType*: WantType     # Note: defaults to enum 0, ie Block
     sendDontHave*: bool     # Note: defaults to false
+    inFlight*: bool         # Whether block sending is in progress. Not serialized.
```

Comment:
We really need an automatic way of handling serde.

Reply:
I hear Eric's made a new library for this. We should take a look.